dev/test/util/helpers.ts

```diff
@@ -89,27 +89,25 @@ export function verifyInstance(firestore: Firestore): Promise<void> {
 }
 
 function write(
-  document: api.IDocument | null,
+  document: api.IDocument,
   mask: api.IDocumentMask | null,
   transforms: api.DocumentTransform.IFieldTransform[] | null,
   precondition: api.IPrecondition | null
 ): api.ICommitRequest {
   const writes: api.IWrite[] = [];
+  const update = Object.assign({}, document);
+  delete update.updateTime;
+  delete update.createTime;
+  writes.push({update});
 
   if (document) {
-    const update = Object.assign({}, document);
-    delete update.updateTime;
-    delete update.createTime;
-    writes.push({update});
     if (mask) {
       writes[0].updateMask = mask;
     }
   }
 
   if (transforms) {
```

Comment:
Same question - could we just remove these if statements?

Reply:
We can't remove them unless we want to also remove the checks in the tests and always include the protos even if they are null/undefined.